<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-NCO37LP" name="com.abc.testng.Launch7" tests="7" failures="0" timestamp="12 Feb 2020 04:25:13 GMT" time="0.020" errors="1">
  <testcase name="test" time="0.001" classname="com.abc.testng.Launch7"/>
  <testcase name="test" time="0.001" classname="com.abc.testng.Launch7"/>
  <testcase name="test" time="0.001" classname="com.abc.testng.Launch7"/>
  <testcase name="logout" time="0.003" classname="com.abc.testng.Launch7"/>
  <testcase name="test" time="0.002" classname="com.abc.testng.Launch7"/>
  <testcase name="login" time="0.011" classname="com.abc.testng.Launch7">
    <error type="java.lang.Error" message="Unresolved compilation problems: 
	Syntax error on token &quot;.&quot;, delete this token
	Type mismatch: cannot convert from void to boolean
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
Syntax error on token ".", delete this token
Type mismatch: cannot convert from void to boolean

at com.abc.testng.Launch7.login(Launch7.java:14)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- login -->
  <testcase name="test" time="0.001" classname="com.abc.testng.Launch7"/>
</testsuite> <!-- com.abc.testng.Launch7 -->
